ReactAndroid/src/main/java/com/facebook/react/modules/network/NetworkingModule.java

```diff
@@ -45,13 +39,13 @@
 import okhttp3.Request;
 import okhttp3.RequestBody;
 import okhttp3.Response;
-import okhttp3.ResponseBody;
 
 /**
  * Implements the XMLHttpRequest JavaScript interface.
  */
 @ReactModule(name = "RCTNetworking", supportsWebWorkers = true)
-public final class NetworkingModule extends ReactContextBaseJavaModule {
+public final class NetworkingModule extends ReactContextBaseJavaModule implements
```

Comment:
@felipecsl Do we actually want to reformat Facebook's code? It makes it harder to see the diff if we ever need to.